### ISSUE TITLE
Where is fetchPlot defined?

### ISSUE THREAD

**Issue by Tooblippe** [hemma/vue-bokeh#1]
Hi,

Were ua the fetchPlot() function defined that is imported?

thanx

**Comment by dylanmcreynolds** [comment]
Indeed, I wonder this, too. The example will not compile without it.

**Comment by hvanova** [comment]
has this issue been resolved?

**Comment by goldentoby** [comment]
Is the issue solved?

**Comment by hemma** [comment]
Hi,
Sorry for the very late response.
I tried finding the original source code but sadly I have changed computers and formatted loads of times since then so none of my backups contained the source.

Im guessing the fetchPlot methods just makes a request to "/get-plot" and set the plot variable. 
If you get it working please make a pull request, for now I will add a "warning" to the readme that it is broken.

**Comment by goldentoby** [comment]
Thank you so much for the reply, I’ve check the code with the backend of python flask.
As you mentioned “/get-plot”, but how can I bridge the backend code with the library “import { fetchPlot } from '../vuex/actions'”

Huge thanks
